code/game/objects/items.dm

```diff
@@ -505,7 +505,7 @@
 					if(automatic)
 						if(H.check_for_open_slot(src))
 							return CANNOT_EQUIP
-					if(H.belt.canremove && !istype(H.belt, /obj/item/weapon/storage/belt))
+					if(H.belt.canremove && !istype(H.belt, /obj/item/weapon/storage/belt) && !istype(H.belt, /obj/item/red_ribbon_arm))
```

Comment:
Does this just disable quick-swapping the Red Ribbon Arm?

Reply:
Yes, that way it will share the functionality that /obj/item/weapon/storage/belt has.

Reply:
That's not a solution. That's a band aid. Fix it properly.